src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 # pylint: disable=line-too-long, consider-using-f-string, no-else-return, duplicate-string-formatting-argument, expression-not-assigned, too-many-locals, logging-fstring-interpolation, broad-except, pointless-statement, bare-except
+from re import match
```

Comment:
Could you please clarify where is this one used?

Reply:
This is not used anywhere with the latest changes so removed it.